tests/sources/git.py

```diff
@@ -695,10 +695,11 @@ def test_track_invalid_submodule(cli, tmpdir, datafiles, fail):
 @pytest.mark.skipif(HAVE_GIT is False, reason="git is not available")
 @pytest.mark.datafiles(os.path.join(DATA_DIR, "template"))
 @pytest.mark.parametrize("ref_format", ["sha1", "git-describe"])
-@pytest.mark.parametrize("tag,extra_commit", [(False, False), (True, False), (True, True)])
-def test_track_fetch(cli, tmpdir, datafiles, ref_format, tag, extra_commit):
+@pytest.mark.parametrize("tag_extra_commit", [(False, False), (True, False), (True, True)])
+def test_track_fetch(cli, tmpdir, datafiles, ref_format, tag_extra_commit):
```

Comment:
In the buildstream repo I disabled `too-many-positional-arguments` in `.pylintrc`. The reason was that the updated `pylint` split `too-many-positional-arguments` from `too-many-arguments`. As we had the latter already disabled, it seemed reasonable to disable the former as well.

Reply:
Nice, that appears to have fixed the linting without my ugly change :)